lips/lip-6.md

```diff
@@ -0,0 +1,88 @@
+---
+lip: 6
+title: Libra Validator Configuration Management
+author: Valeria Nikolaenko, David Wolinsky, Sam Blackshear, David Dill
+status: Draft
+type: Informational
+created: 07/30/2020
+---
+
+# Summary
+---
+This LIP describes the conceptual model and implementation of validator configuration management on the Libra blockchain.
+
+---
+# Abstract / Motivation
+---
+
+The Libra Payment Network (“LPN”) is a blockchain-backed payment platform and an infrastructure for stablecoins that supports a regulated global digital payment network. Both the blockchain and the stablecoin financial instruments are controlled by Libra Networks, Ltd. (“Libra Networks” or “LN”) a subsidiary of the Libra Association  (the “Association” or “LA”).  LPN operates under a payment system license from FINMA (the “FINMA License”), and LN is responsible for ensuring that LPN operates in compliance with the FINMA License.
+
+The Libra Association is an independent membership organization that is responsible for the governance of the LPN and the development of the Libra project. The Association is governed by the Council, which is comprised of one representative per Association member. The Association Council manages the technology roadmap of the platform and oversees and maintains the Libra Improvement Proposal and Libra Upgrade Process, often through subcommittees of the Council such as the Technical Steering Committee (the “TSC”) and the Association Board. 
+
+Each member that participates in operating the blockchain by operating a Validator Node is referred to as a Validator Owner. The Validator Set defines the current set of Validator Nodes that run consensus protocol. Validator Nodes that are currently part of the Validator Set process transactions and interact with each other to reach consensus on the state of the blockchain. The Validator Set may change from time to time and not all members may have Validator Nodes the Validator Set at any given time. Reasons for changing the Validator Set include joining or departure of members from the Association and actions needed for compliance reasons or to comply with LN’s obligations under its payment system license from FINMA to guarantee the secure and performant operation of the platform.
+
```

Comment:
Could we consistently use throughout:
Validator Owner (capitals)
Validator Node
Validator Operator

I also wonder whether on-chain, we could change validator accounts to be named validator-owner accounts?

Reply:
Great suggestion! I tried to make sure the terminology is consistent and I also added a section Terminology for better clarity.

We could rename on-chain: validator_role to validator_owner_role and rename the script create_validator_account to create_validator_owner account. I added a TODO here: https://github.com/libra/libra/issues/5117